src/Packagist/WebBundle/Entity/Version.php

```diff
@@ -179,6 +179,11 @@ class Version
      */
     private $support;
 
+    /**
+     * @ORM\Column(type="text", nullable=true)
```

Comment:
As per discussion this should be type json, then the set/getter can be simplified